update LocalStack CLI instructions (#668)
localstack/docs: content/en/getting-started/quickstart.md

Reviewer: there is a gap here between install and quickstart. we end the install with `localstack --version` and then continue with assuming the api key is already configered `After installing LocalStack and configuring your API key, let us start LocalStack. To run our sample AWS application, we will start LocalStack using the localstack CLI.`

a quick fix would be to change the order in the hierarchy (install -> api key -> quickstart) and also pointing to the api key after the install updating this section `You are all set! Check out our [Quickstart guide](https://localstack-docs-preview-pr-668.surge.sh/getting-started/quickstart/) to start your local cloud!`

Author: The last commit reorders the page (install -> api key -> quick start) and slightly changes the "you are all set" sentence. :)

You can optionally run your LocalStack container in background mode by adding the `-d` flag to the `localstack start` command.

{{< alert title="Notes" >}}
- This command loads all services provided by LocalStack, they will however be started on the first request reaching this service.

- By default, LocalStack uses the image tagged `latest` that is cached on your machine, and will **not** pull the latest image automatically from Docker Hub (i.e., the image needs to be pulled manually if needed).

> - The default image `localstack/localstack` in DockerHub refers to the community version of LocalStack.

Reviewer 2: ```suggestion
- The default image [`localstack/localstack`](https://hub.docker.com/r/localstack/localstack) on Docker Hub refers to the community version of LocalStack.
```

>   The `localstack/localstack-pro` image refers to the Pro version of LocalStack. Previously we maintained `localstack-light` and `localstack-full` images which has been deprecated and will be removed with the LocalStack 2.0 release.

Reviewer 2: ```suggestion
  The [`localstack/localstack-pro`](https://hub.docker.com/r/localstack/localstack-pro) image refers to the Pro version of LocalStack. Previously we maintained `localstack-light` and `localstack-full` images which have been deprecated and removed with the LocalStack 2.0 release.
```

> - When an [API key]({{< ref "api-key" >}}) is set in your environment, `localstack start` will attempt to start LocalStack with the `localstack/localtack-pro` image.

Reviewer 2: ```suggestion
- If an [API key]({{< ref "api-key" >}}) is set in your environment, `localstack start` will attempt to start LocalStack with the `localstack/localtack-pro` image.
```
